CMakeLists.txt

```diff
@@ -226,6 +228,7 @@ if (NOT TARGET Kokkos::kokkos)
 endif()
 
 # Build Tests and download Catch2
+set(PARTHENON_TEST_DIR ${CMAKE_CURRENT_SOURCE_DIR}/tst/regression CACHE INTERNAL "Absolute path dir Parthenon regression test directory.")
```

Comment:
I would change CMAKE_CURRENT_SOURCE_DIR to PARTHENON_SOURCE_DIR.

Reply:
Actually, I think it has to be `CMAKE_CURRENT_SOURCE_DIR`. The intent here is for downstream codes to be able to build tests too.

Reply:
I don't think that is accurate, because he is using PARTHENON_TEST_DIR to point to the run_test.py script, which will always be in the Parthenon repo if I'm not mistaken.

Reply:
You're both right and both versions did achieve this. The important piece was that it's an absolute path to the `run_test.py` provided by `Parthenon`.
I now removed that variable entirely and directly use the `parthenon_SOURCE_DIR` in `TestSetup.cmake`